CIScripts/Test/Tests/Protocol/SingleHostWithUtils.Tests.ps1

```diff
@@ -118,9 +101,45 @@ Describe "Single compute node protocol tests with utils" {
             -TenantName $ControllerConfig.DefaultProject `
             -Name $NetworkName `
             -Subnet "$( $Subnet.IpPrefix )/$( $Subnet.IpPrefixLen )"
+
+        Write-Host "Creating containers"
+        $Container1ID, $Container2ID = Invoke-Command -Session $Session -ScriptBlock {
+            docker run --network $Using:NetworkName -d iis-tcptest ping -t localhost
+            docker run --network $Using:NetworkName -d microsoft/nanoserver ping -t localhost
+        }
+
+        Write-Host "Getting VM NetAdapter Information"
+        $VMNetInfo = Get-RemoteNetAdapterInformation -Session $Session `
+            -AdapterName $SystemConfig.AdapterName
+
+        Write-Host "Getting vHost NetAdapter Information"
+        $VHostInfo = Get-RemoteNetAdapterInformation -Session $Session `
+            -AdapterName $SystemConfig.VHostName
+
+        Write-Host "Getting Containers NetAdapter Information"
+        $Container1NetInfo = Get-RemoteContainerNetAdapterInformation `
+            -Session $Session -ContainerID $Container1ID
+        $Container2NetInfo = Get-RemoteContainerNetAdapterInformation `
+            -Session $Session -ContainerID $Container2ID
+
+        Initialize-ContainersConnection -VMNetInfo $VMNetInfo -VHostInfo $VHostInfo `
+            -Container1NetInfo $Container1NetInfo -Container2NetInfo $Container2NetInfo `
+            -Session $Session
+
+        Write-Host "Executing netsh"
```

Comment:
This code doesn't seem to have an equivalent in removed/changed code. Is there some piece of dead code somewhere that was responsible for setting up arp table?